documentation/how-to-branch.md

```diff
@@ -1,96 +1,85 @@
-# This will show you how you will branch from your respective work branch. 
+# Branching Guide for Development Teams
 
-Each team has their own main working branch `FE-Offers` `FE-Listing` etc. 
-From this main working branch each dev will make the branch for the task their working on. 
+## Overview
 
-ex: 
-Ola Nordmann is working on creating a listener for createOffer. Ola will then create a branch from `FE-Offers` that is named 
-```
-FE-Offers-createOfferListener
-```
-and then add the file for the listener in that branch. 
-This will give a more sense of ownership, easier for the general overview when you know who has been working on something,
-rather then spending time figure out who has worked on what files. 
+It's recommended for each development team to have its own main working branch (e.g., `teamName-Offers`, `teamName-Listing`). Developers are encouraged to create task-specific branches from their team's main working branch for their individual tasks. However, it's important to consult with the team leads or teachers before proceeding to ensure alignment with the overall project strategy.
 
-![A graphic displaying the branching format](https://i.imgur.com/vdOzdpk.png)
+### Important Notes:
 
+- Avoid working directly in the main/master branch. Always use your team's branch.
```

Comment:
Main is a protected branch, so it cannot be worked with directly without the proper permissions.
Maby switch up the statement "Avoid work..." with something more reflective about main being protected, and all PR needs minimum 1 review before a merge is accepted ?

Reply:
Better?